hstest/testing/plotting/seaborn_handler.py

```diff
@@ -0,0 +1,203 @@
+from .drawing import Drawing
+from importlib import reload
+from hstest.testing.plotting.matplotlib_handler import MatplotlibHandler
+
+
+class SeabornHandler:
+    _saved = False
+    _replaced = False
+
+    _displot = None
+    _histplot = None
+    _lineplot = None
+    _lmplot = None
+    _scatterplot = None
+    _catplot = None
+    _barplot = None
+    _violinplot = None
+    _heatmap = None
+    _boxplot = None
+
+    @staticmethod
+    def replace_plots(drawings):
+        try:
+            import seaborn as sns
+        except ModuleNotFoundError:
+            return
+
+        lib_type = 'seaborn'
+
+        def displot(data=None, **kwargs):
+            drawing = Drawing(
+                lib_type,
+                'displot',
```

Comment:
What is this type of plot? Does it exist in Matplotlib? Considering that Seaborn uses Matplotlib to draw graphics, maybe there is some plot like that in Matplotlib also?